dstk-infra/apollo/src/graphql/storage-provider/storageProvider.ts

```diff
@@ -35,4 +36,15 @@ export class ObjectionStorageProvider extends Model {
     static get idColumn() {
         return 'providerId';
     }
+
+    static relationMappings = () => ({
+        models: {
+            relation: Model.HasManyRelation,
+            modelClass: ObjectionMLModel,
+            join: {
+                from: 'registryStorageProviders.id',
```

Comment:
Could you verify the behavior here? I don't know if it will reference the literal `id` column or the aliased one.

Either way it should be `to: registryModels.storageProviderId`, right?